### ISSUE TITLE
Un-blocking all scripts one-by-one still breaks a website until the general Block Scripts is disabled

### ISSUE THREAD

**Issue by smartfonreddit** [brave/brave-browser#1358]
## Description 
_"Toggle switches to disable script blocking"_ section apparently doesn't list all the blocked scripts. Disabling the blocking of all the listed scripts one-by-one still breaks a website. The only way to un-break is to disable the general "Block scripts" switch.

Image: https://i.imgur.com/V4AjxMh.png

## Steps to Reproduce

   1.  Enable the global Block Scripts from Settings.
   2. On this page, the video at the top is broken https://www.independent.co.uk/news/world/europe/macedonia-name-change-referendum-north-greece-dispute-nato-eu-european-union-membership-a8562071.html
   3. While leaving the global "Block scripts" switch enabled, flip all script switches below to allow all scripts individually. Click Apply.

## Actual result:
The video is still broken. To fix it, disable the general "Block scripts" switch above.

## Expected result:
After un-blocking all scripts individually, it should be the same as disabling "Block Scripts" for that page, yet "Block Scripts" also blocks scripts that aren't listed in the list of blocked scripts.

## Reproduces how often: 
Easily reproduced

## Brave version (chrome://version info)

Brave | 0.55.10 Chromium: 70.0.3538.22 (Official Build) beta (64-bit)
-- | --
Revision | ac9418ba9c3bd7f6baaffa0b055dfe147e0f8364-refs/branch-heads/3538@{#468}
OS | Windows

**Comment by smartfonreddit** [comment]
On this next example page, it looks like the shield popup displays all the blocked scripts, it's just that toggling to enable a script will make the switch grey, yet the script won't be allowed through. 

Try: https://www.bleepingcomputer.com/news/google/google-maps-is-getting-a-big-update-with-commute-tab-and-more/ 

Below the bold text **"Mixed-mode commutes support"** there is an image that's  supposed to be loaded from bleepstatic.com (the website's CDN). If I manually enable bleedstatic.com from the Shield and refresh the page, the image still won't load. If I disable the Block Scripts globally, then it loads. Looks like the individual script toggles are broken for _some_ scripts, but work on others.

Another issue is that closing and re-opening that page resets the individual script blocking settings. I have to toggle it for bleepstatic.com again every time I open the page. This might be the "bigger" problem.

**Comment by tomlowenthal** [comment]
The screenshot looks like the older shields UI. Does the same thing happen in the newer midnight-colored Shields UI?

**Comment by smartfonreddit** [comment]
Still happening, but the area that shows all the blocked scripts now show an entry named "**null/**" even after unblocking all scripts.

https://i.imgur.com/Jglm27W.png

No matter how many times I "Undo" null/'s blockage, or click Allow All to allow it, the null/ keeps returning after the page refresh.

Also, the browser doesn't remember that I allowed all scripts for that page after closing the tab. All the previously white-listed scripts are blocked again after re-visiting the page.

**Comment by tomlowenthal** [comment]
Individual script choices currently only get applied once, not persistently — that's why the button says "apply once". Fixing that is #1889.

If you're seeing a "null/" item, that makes me think this is a dupe of #1901. If this issue is still around when that bug's fixed, comment here and I'll re-open this so that we can look into this further.
